Fix test failure against devel

It looks like a test is failing against devel due to a new variable in the combined data:

=================================== FAILURES ===================================
____________________ test_data_processing_surface_notracer _____________________

tac_ch4_data_args = {'averaging_period': ['1h'], 'bc_store': 'inversions_tests', 'domain': 'EUROPE', 'emissions_name': ['total-ukghg-edgar7'], ...}
raw_data_path = PosixPath('/home/runner/work/openghg_inversions/openghg_inversions/tests/data')
using_zarr_store = True, openghg_version = (0, 14, 0)

    def test_data_processing_surface_notracer(
        tac_ch4_data_args, raw_data_path, using_zarr_store, openghg_version
    ):
        """Check that `data_processing_surface_notracer` produces the same output
        as v0.1, with test data frozen on 9 Feb 2024, or the same as v0.2, with test data frozen on
        15 Apr 2024 (using the zarr backend).
        """
        result = data_processing_surface_notracer(**tac_ch4_data_args)
    
        # check number of items returned
        assert len(result) == 6
    
        # check keys of "fp_all"
        assert list(result[0].keys()) == [".species", ".flux", ".bc", "TAC", ".scales", ".units"]
    
        if openghg_version >= (0, 13):
            # get combined scenario for TAC at time 2019-01-01 00:00:00; "frozen" data made
            # with OpenGHG 0.13
            ds = xr.open_dataset(raw_data_path / "merged_data_test_tac_combined_scenario_v13.nc")
            expected_tac_combined_scenario = fp_all_from_dataset(ds)
    
>           xr.testing.assert_allclose(
                result[0]["TAC"].isel(time=0).load(), expected_tac_combined_scenario["TAC"].isel(time=0)
E               AssertionError: Left and right Dataset objects are not close
E               Data variables only on the left object:
E                   release_height                       float64 8B 185.0

--- a/tests/test_get_data.py
+++ b/tests/test_get_data.py
@@ -36,6 +36,9 @@ def test_data_processing_surface_notracer(
     # check keys of "fp_all"
     assert list(result[0].keys()) == [".species", ".flux", ".bc", "TAC", ".scales", ".units"]
 
+    # variables to check (to avoid surprises from new variables added to data)
+    check_vars = ["mf", "fp", "mf_mod", "bc_mod"]
+
     if openghg_version >= (0, 13):
         # get combined scenario for TAC at time 2019-01-01 00:00:00; "frozen" data made
         # with OpenGHG 0.13
@@ -43,7 +46,7 @@ def test_data_processing_surface_notracer(
         expected_tac_combined_scenario = fp_all_from_dataset(ds)
 
         xr.testing.assert_allclose(
-            result[0]["TAC"].isel(time=0).load(), expected_tac_combined_scenario["TAC"].isel(time=0)
+            result[0]["TAC"][check_vars].isel(time=0).load(), expected_tac_combined_scenario["TAC"][check_vars].isel(time=0)
         )
     elif using_zarr_store:
         # get combined scenario for TAC at time 2019-01-01 00:00:00

